agents/cdu/fence_cdu.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/python3
+#    fence_cdu - fence agent for a Sentry Switch CDU.
+#
+#    Copyright (C) 2012 Canonical Ltd.
+#    Copyright (C) 2021 SUSE Linux GmbH <trenn@suse.de>
+#
+#    Authors: Andres Rodriguez <andres.rodriguez@canonical.com>
+#             Thomas Renninger <trenn@suse.de>
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU General Public License as published by
+#    the Free Software Foundation, version 3 of the License.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU General Public License for more details.
+#
+#    You should have received a copy of the GNU General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+#####
+##
+## The Following Agent Has Been Tested On:
+##
+##  Model                Firmware
+## +---------------------------------------------+
+## Sentry Switched CDU   6a
+## Sentry Switched CDU   7.1c <trenn@suse.de>
+##
+## @note: ssh has not being tested
+##
+#####
+
+import sys, re, pexpect, atexit
+sys.path.append("/usr/share/fence")
```

Comment:
This also needs to be changed to `@FENCEAGENTSLIBDIR@` to get replaced with detected/specified path when you build the project.